blackbox/docs/sql/reference/create_ingest_rule.txt

```diff
@@ -0,0 +1,86 @@
+.. _create-ingest-rule:
+
+======================
+``CREATE INGEST RULE``
+======================
+
+Defines a new ingestion rule.
+
+Synopsis
+========
+.. highlight:: psql
+
+::
+
+    CREATE INGEST RULE rule_name
+    ON source_ident
+    [WHERE condition]
+    INTO table_ident
+
+
+Description
+===========
+
+``CREATE INGEST RULE`` creates a new ingestion rule, applying on a
+``source_ident`` and ingesting into the specified target ``table_ident``.
+
+The ingestion rules will be used by ingestion plugins to filter the data
+streams of its ``source_ident``, the filtered data will then be
+stored in the corresponding target ``table_ident``.
+
+Parameters
+==========
+
+:rule_name: The identifier of the rule to create.
+
+:source_ident: The registration identifier of an ingestion source.
+
+:condition: A boolean expression using references specific to the ingest
+            source.
+
+:table_ident: The identifier of the target table.
+
+
+Notes
+=====
+
+.. NOTE::
+
+    When the :ref:`administration_user_management` is enabled, this statement
+    can only be issued by a superuser.
+
+    The default user for the ``INSERT`` operations in the target
+    ``table_ident`` is the superuser ``crate``.
+
+If no ingestion plugin is registered, the ingestion rules will not take any
+effect.
+
+There is no default ingestion rule, all incoming messages into any ingestion
+endpoint are discarded by default.
+
+The references to be used in the ``WHERE`` clause are specific to each
+ingestion plugin.
+
```

Comment:
we should point out that the target table must be created manually and should contain the same columns of the plugins data structure. otherwise the ingestion fails and an error is logged